### PR TITLE
Add  CC_TAOBAO

### PR DESCRIPTION
**Pull request by yyycggg**
Re: https://github.com/cocos/3d-tasks/issues/14417

### Changelog

*

-------

### Continuous Integration

This pull request:

* [ ] needs automatic test cases check.
  > Manual trigger with `@cocos-robot run test cases` afterward.
* [ ] does not change any runtime related code or build configuration
  > If any reviewer thinks the CI checks are needed, please uncheck this option, then close and reopen the issue.

-------

### Compatibility Check

This pull request:

* [ ] changes public API, and have ensured backward compatibility with [deprecated features](https://github.com/cocos/cocos-engine/blob/v3.5.0/docs/contribution/deprecated-features.md).
* [ ] affects platform compatibility, e.g. system version, browser version, platform sdk version, platform toolchain, language version, hardware compatibility etc.
* [ ] affects file structure of the build package or build configuration which requires user project upgrade.
* [ ] **introduces breaking changes**, please list all changes, affected features and the scope of violation.

### CODE REVIEW

**Comment by minggo** [comment]
Where is `CC_TAOBAO`?

**Comment by yyycggg** [comment]
> Where is `CC_TAOBAO`?

Add `ccGlobal : true`, and the `CC_TAOBAO` will be exported when packaging.

**Review by minggo — APPROVED** [review]

**Comment by minggo** [comment]
please fix npm test error.

File: cc.config.json [hunk 1]
```diff
@@ -323,6 +323,7 @@
             "comment": "Running in the taobao creative app.",
             "type": "boolean",
             "value": false,
+            "ccGlobal": true,
```
Comment (PPpro) [inline cc.config.json:326]: Why add `CC_TAOBAO` ?

**Comment by PPpro** [comment]
We don't add `CC_XXX` macro for new platforms.
See https://github.com/cocos/3d-tasks/issues/14417#issuecomment-1321470850

**Pull request closed without merging**